app/src/pages/inside/suitesPage/suitesPage.jsx

```diff
@@ -36,254 +36,211 @@ import {
-    itemCount: PropTypes.number,
-    pageCount: PropTypes.number,
-    pageSize: PropTypes.number,
-    sortingColumn: PropTypes.string,
-    sortingDirection: PropTypes.string,
-    fetchTestItemsAction: PropTypes.func,
-    onChangePage: PropTypes.func,
-    onChangePageSize: PropTypes.func,
-    onChangeSorting: PropTypes.func,
-    toggleSuiteSelectionAction: PropTypes.func,
-    unselectAllSuitesAction: PropTypes.func,
-    toggleAllSuitesAction: PropTypes.func,
-    selectSuitesAction: PropTypes.func,
-    parentItem: PropTypes.object,
-    loading: PropTypes.bool,
-    validationErrors: PropTypes.object,
-    onFilterAdd: PropTypes.func,
-    onFilterRemove: PropTypes.func,
-    onFilterValidate: PropTypes.func,
-    onFilterChange: PropTypes.func,
-    filterErrors: PropTypes.object,
-    filterEntities: PropTypes.array,
-    tracking: PropTypes.shape({
-      trackEvent: PropTypes.func,
-      getTrackingData: PropTypes.func,
-    }).isRequired,
-    highlightItemId: PropTypes.number,
-  };
+export const SuitesPageWrapped = ({
+  deleteItems,
+  onEditItem,
+  onEditItems,
+  activePage,
+  itemCount,
+  pageCount,
+  pageSize,
+  sortingColumn,
+  sortingDirection,
+  onChangePage,
+  onChangePageSize,
+  onChangeSorting,
+  onFilterAdd,
+  onFilterRemove,
+  onFilterValidate,
+  onFilterChange,
+  filterErrors,
+  filterEntities,
+}) => {
+  const { trackEvent } = useTracking({ page: SUITE_PAGE });
 
-  static defaultProps = {
-    deleteItems: () => {},
-    onEditItem: () => {},
-    onEditItems: () => {},
-    suites: [],
-    selectedSuites: [],
-    activePage: DEFAULT_PAGINATION[PAGE_KEY],
-    itemCount: null,
-    pageCount: null,
-    pageSize: DEFAULT_PAGINATION[SIZE_KEY],
-    sortingColumn: null,
-    sortingDirection: null,
-    fetchTestItemsAction: () => {},
-    onChangePage: () => {},
-    onChangePageSize: () => {},
-    onChangeSorting: () => {},
-    toggleSuiteSelectionAction: () => {},
-    unselectAllSuitesAction: () => {},
-    toggleAllSuitesAction: () => {},
-    selectSuitesAction: () => {},
-    parentItem: null,
-    loading: false,
-    validationErrors: {},
-    onFilterAdd: () => {},
-    onFilterRemove: () => {},
-    onFilterValidate: () => {},
-    onFilterChange: () => {},
-    filterErrors: {},
-    filterEntities: [],
-    highlightItemId: null,
+  const [highlightedRowId, setHighlightedRowId] = useState(null);
+  const [isGridRowHighlighted, setIsGridRowHighlighted] = useState(false);
+  const debugMode = useSelector(debugModeSelector);
+  const suites = useSelector(suitesSelector);
+  const selectedSuites = useSelector(selectedSuitesSelector);
+  const parentItem = useSelector(parentItemSelector);
+  const loading = useSelector(loadingSelector);
+  const validationErrors = useSelector(validationErrorsSelector);
+  const highlightItemId = useSelector(prevTestItemSelector);
+  const userRoles = useSelector(userRolesSelector);
+  const canSelectItems = canManageTestItemsActions(userRoles);
```

Comment:
Maybe left more generic name here? In case it will manage other types of actions in this component in the future?